<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlobWidget</class>
 <widget class="QWidget" name="BlobWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>640</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextPage">
   <property name="geometry">
    <rect>
     <x>926</x>
     <y>510</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_CalcResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>498</y>
     <width>640</width>
     <height>171</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <column>
    <property name="text">
     <string>面积</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>当前角度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>长宽比</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>宽度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>长度</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>895</x>
     <y>521</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrePage">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>510</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>10</y>
     <width>351</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_11">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>150</y>
       <width>341</width>
       <height>331</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>筛选条件</string>
     </property>
     <widget class="QGroupBox" name="groupBox_BlobRatio">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>170</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>长宽比</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="txtThresholdRatioLow">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtThresholdRatioHigh">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>95</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>44</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_SingalArea">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>291</width>
        <height>131</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>面积</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLineEdit" name="txtThresholdHigh">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtThresholdLow">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>90</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>43</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>94</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>连通域排序</string>
     </property>
     <widget class="QComboBox" name="comboBoxSort">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>不排序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X坐标升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>X坐标降序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y坐标升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Y坐标降序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>上到下，左到右</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>左到右，上到下</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>面积升序</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>面积降序</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>检测个数</string>
     </property>
     <widget class="QLineEdit" name="txtNum">
      <property name="geometry">
       <rect>
        <x>26</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_AngleLock">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>角度锁定</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>10</y>
       <width>341</width>
       <height>461</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>判决条件</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QGroupBox" name="groupBox_BlobCount">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>有效blob数量</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_BlobCount">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>85</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBlobCountMax">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBlobCountMin">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>124</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>44</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>85</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>计算值</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_BlobArea">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>200</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="title">
       <string>有效blob面积总数</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_SumArea">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>85</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBlobAreaMax">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBlobAreaMin">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>---</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>45</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>上限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>125</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>下限</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>86</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>计算值</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_AllIndex">
   <property name="geometry">
    <rect>
     <x>908</x>
     <y>521</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CurrentIndex">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>521</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>计算门限</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btnAutoCalc">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>计算</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_AutoSetResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>自动设置</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>648</x>
     <y>10</y>
     <width>3</width>
     <height>480</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
